assets/images/getting-started/number-1.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="50px" height="50px" viewBox="0 0 50 50" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49 (51002) - http://www.bohemiancoding.com/sketch -->
+    <title>Number</title>
+    <desc>Created with Sketch.</desc>
+    <defs>
+        <circle id="path-1" cx="25" cy="25" r="25"></circle>
+    </defs>
+    <g id="Getting-started" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Bitcoin.org---Getting-Started--Desktop" transform="translate(-775.000000, -724.000000)">
```

Comment:
Metadata of the SVGs contain references to Bitcoin.org and btc - I think se should look to create our own assets here instead, these are fairly simple.

Reply:
See above answer.